### PR TITLE
Feature/umbraco7

### PR DESCRIPTION
**Pull request by msoelvsteen**

### CODE REVIEW

File: src/Enterspeed.Source.UmbracoCms.V7/Data/Repositories/EnterspeedJobRepository.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    {
+        private readonly string _tableName = "EnterspeedJobs";
+        public IList<EnterspeedJob> GetFailedJobs()
+        {
+            var db = ApplicationContext.Current.DatabaseContext.Database;
+
+            var failedJobs = db.Fetch<EnterspeedJobSchema>(
+                $"SELECT * FROM [{_tableName}] WHERE [JobState] = {EnterspeedJobState.Failed.GetHashCode()} ORDER BY [CreatedAt]");
+
+            var result = failedJobs.Select(MapFromSchema).Where(x => x != null).ToList();
```
Comment (Malthebp) [inline src/Enterspeed.Source.UmbracoCms.V7/Data/Repositories/EnterspeedJobRepository.cs:20]: Are we sure that db.Fetch<> cannot return null?

File: src/Enterspeed.Source.UmbracoCms.V7/Services/EnterspeedJobService.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            _contentService = ApplicationContext.Current.Services.ContentService;
+            _enterspeedJobRepository = EnterspeedContext.Current.Repositories.JobRepository;
+        }
+
+        public SeedResponse Seed()
+        {
+            var allContent = new List<IContent>();
+
+            // Add all root nodes
+            var rootContent = _contentService.GetRootContent().ToList();
```
Comment (Malthebp) [inline src/Enterspeed.Source.UmbracoCms.V7/Services/EnterspeedJobService.cs:32]: We are only getting published content here, right?

File: src/Enterspeed.Source.UmbracoCms.V7/Data/Repositories/EnterspeedJobRepository.cs (hunk 1 above)
Comment (msoelvsteen) [reply src/Enterspeed.Source.UmbracoCms.V7/Data/Repositories/EnterspeedJobRepository.cs]: It shouldn't, but I've updated it so it handles if it returns null

File: src/Enterspeed.Source.UmbracoCms.V7/Services/EnterspeedJobService.cs (hunk 2 above)
Comment (msoelvsteen) [reply src/Enterspeed.Source.UmbracoCms.V7/Services/EnterspeedJobService.cs]: No, actually we got content from db. I've changed so it seeds from the Umbraco cache instead

**Review by Malthebp — APPROVED** [review]

**Review by Malthebp — APPROVED** [review]

**Pull request merged**